app/views/layouts/view_data.html.erb

```diff
@@ -4,7 +4,11 @@
     <%= tag_manager_js -%>
     <meta charset="utf-8">
 
-    <title><%= page.title %></title>
+    <% if @settings && @settings.invalid? %>
```

Comment:
Next time we need to do this, we can generalise in the application_controller. But okay for now.